app/controllers/api/v1/people/people_controller.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# Api::V1 module
+module Api::V1
+  # People::PeopleController controller
+  class People::PeopleController < ApiController
+    before_action :authenticate_v1_user!, except: %i[index show]
```

Comment:
Requirements demand authenticated users for "unsafe methods". Here need to add them to `:authenticate_v1_user!` before_action. (same for MoviesController)